cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/configurations/RequestConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package com.farao_community.farao.cse.import_runner.app.configurations;
+
+import com.farao_community.farao.cse.import_runner.app.services.RequestService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import reactor.core.publisher.Flux;
+
+import java.util.function.Function;
+
+@Configuration
+public class RequestConfiguration {
+
+    @Autowired
+    RequestService requestService;
```

Comment:
It is advised by Spring to use constructor injection instead

Reply:
Corrected